scripts/persistent-docker-setup/detect_ips.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Fetch IPv4 WAN address using ifconfig.me, fallback to ipinfo.io
```

Comment:
If this script is also here and the node-installer then I think we can do something to remove duplication?

Reply:
Okay, done.